### PR TITLE
If the framebuffer attachment isn't available, log the reason.

### PR DESCRIPTION
**Pull request by ben-clayton**
Fixes: #603

### CODE REVIEW

File: gapis/api/gles/replay.go [hunk 1]
```diff
@@ -275,7 +275,7 @@ func (t *destroyResourcesAtEOS) Flush(ctx context.Context, out transform.Writer)
 		// change to the gles.api file.
 
 		// Delete all Renderbuffers.
-		renderbuffers := make([]RenderbufferId, 0, len(c.Objects.Shared.Renderbuffers)-3)
+		renderbuffers := make([]RenderbufferId, 0, len(c.Objects.Shared.Renderbuffers))
```
Comment (ben-clayton) [inline gapis/api/gles/replay.go]: Unrelated fix for a panic. If there's no shared renderbuffers, then you'll get a negative cap.

File: gapis/api/gles/read_framebuffer.go [hunk 2]
```diff
@@ -53,11 +53,12 @@ func (t *readFramebuffer) Transform(ctx context.Context, id api.CmdID, cmd api.C
 
 func (t *readFramebuffer) Flush(ctx context.Context, out transform.Writer) {}
 
-func (t *readFramebuffer) Depth(id api.CmdID, res replay.Result) {
+func (t *readFramebuffer) Depth(ctx context.Context, id api.CmdID, res replay.Result) {
```
Comment (dsrbecky) [inline gapis/api/gles/read_framebuffer.go]: Why the extra arg?
Muscle memory?

Comment (ben-clayton) [inline gapis/api/gles/read_framebuffer.go]: So we can log...

Comment (dsrbecky) [inline gapis/api/gles/read_framebuffer.go]: Isn't it unused?  (shadowed line below)

Comment (ben-clayton) [inline gapis/api/gles/read_framebuffer.go]: ewww, yes. Okay, will remove
